src/index.html

```diff
@@ -2,11 +2,50 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="content">
+      <section class="calendar calendar--start-day--fri calendar--month-end--31">
+        <div class="calendar__days"></div>
```

Comment:
My assumption class name should be `calendar__day`
```suggestion
        <div class="calendar__day"></div>
```